all/pom.xml

```diff
@@ -190,11 +190,12 @@
             <artifactId>seata-sqlparser-core</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <!-- Slimming down for 'seata-all.jar'
         <dependency>
             <groupId>io.seata</groupId>
             <artifactId>seata-sqlparser-antlr</artifactId>
             <version>${project.version}</version>
-        </dependency>
+        </dependency>-->
```

Comment:
If the user is configured to use `antlr`, is there a friendly prompt for the user?

Reply:
done

Reply:
the code is removed to the PR #4353